web/api/formatters/rrd2json.c

```diff
@@ -95,8 +94,6 @@ void build_context_param_list(struct context_param **param_list, RRDSET *st)
         memcpy(rd->state, rd1->state, sizeof(*rd->state));
         memcpy(&rd->state->collect_ops, &rd1->state->collect_ops, sizeof(struct rrddim_collect_ops));
         memcpy(&rd->state->query_ops, &rd1->state->query_ops, sizeof(struct rrddim_query_ops));
-        rd->state->metric_uuid = mallocz(sizeof(uuid_t));
-        uuid_copy(*rd->state->metric_uuid, *rd1->state->metric_uuid);
```

Comment:
Not sure I understand why we need to delete this.

Reply:
The `memcpy` will copy the `rd->state` which includes the `state->metric_uuid`